### DISCUSSION TITLE
Official Drupal theme?

### DISCUSSION THREAD

**Discussion by patrickcate** [uswds/uswds#4768] [General]
Given the prevalence of Drupal as a CMS for U.S. Government websites, I was wondering if there were any plans on developing an official USWDS Drupal theme? Currently there are two different contributed Drupal themes, but I'm wondering if an official theme would help unify and focus all USWDS Drupal theme development. With USWDS 3.0 now using Twig with Storybook, it seems to LOE would be lower than in past versions.

**Comment by mejiaj** [comment]
Hey @patrickcate, sorry but this isn't something that's on our [Roadmap](https://designsystem.digital.gov/about/product-roadmap/). We're happy to collaborate and provide feedback, but we can't take on a big project like this.

Additionally, there are some differences between the version of twig we use and the one Drupal uses. For example, we don't have access to drupal twig filters to set attributes. So getting feature parity there would require additional work.

**Comment by mgifford** [comment]
The two Drupal themes are https://www.drupal.org/project/uswds & https://www.drupal.org/project/uswds_base

I've talked with our team at CivicActions about combining issues in the past. It does take more time though in the short term so we'd likely need to run this by our clients to see that there is interest in supporting it. 

Would be nice to have a tighter integration of best practices between the design system & popular CMS. There are people in the Drupal community looking at adopting best practices like web components too. Not an issue for 3.0, but likely something that may be an issue for the next major release. 

Most of the time clients don't want to pay money now for something that will save them money in the future. Consolidating efforts fits into that.
